src/CLI/Command/Init.php

```diff
@@ -0,0 +1,83 @@
+<?php
+declare(strict_types=1);
+
+namespace Arkitect\CLI\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Webmozart\Assert\Assert;
+
+class Init extends Command
+{
+    public static $defaultName = 'init';
+
+    public static $defaultDescription = <<< 'EOT'
+Creates a new phparkitect.php file
+EOT;
+
+    public static $help = <<< 'EOT'
+This command creates a new phparkitect.php in the current directory
+You can customize the directory where the file is created specifying <comment>-d /dest/path</comment>
+EOT;
+
+    protected function configure(): void
+    {
+        $this
+            ->addUsage('creates a phparkitect.php file in the current dir')
+            ->addUsage('--dest-dir=/path/to/dir creates a phparkitect.php file in /path/to/dir')
+            ->addUsage('-d /path/to/dir creates a phparkitect.php file in /path/to/dir')
+            ->setHelp(self::$help)
+            ->addOption(
+                'dest-dir',
+                'd',
+                InputOption::VALUE_REQUIRED,
+                'destination directory for the file',
+                '.'
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $output->writeln('');
+
+        try {
+            $sourceFilePath = __DIR__.'/../../../phparkitect-stub.php';
```

Comment:
Have you tried launching the command with `.phar`?
In past, we had problems with path when we try to run a command with our different installations: `composer` and `.phar`

Reply:
Nope, i'll give it a try

Reply:
yup, phar generation needed a fix, everything should be fine now